java-extensions/hudi-reader/pom.xml

```diff
@@ -16,7 +16,9 @@
         <maven.compiler.target>8</maven.compiler.target>
         <java-extensions.home>${basedir}/../</java-extensions.home>
         <slf4j.version>1.7.32</slf4j.version>
-        <hudi.version>0.11.1</hudi.version>
+        <presto.hadoop.version>2.7.4-11</presto.hadoop.version>
+        <presto.hive.version>3.0.0-8</presto.hive.version>
+        <hudi.version>0.12.1</hudi.version>
```

Comment:
The above attributes (hadoop/hive) are not used and should keep the HUDI version